package.json

```diff
@@ -16,6 +16,7 @@
     "prettier:check": "exec prettier . --check"
   },
   "dependencies": {
+    "@calimero-network/calimero-client": "1.5.0",
```

Comment:
This is waiting for [this](https://github.com/calimero-network/calimero-client-js/pull/37/) PR?